File: Challenges/4 - Whereareyou/Whereareyou/app/src/main/java/com/example/whereareyou/MainActivity.java
package com.example.whereareyou;
import android.content.Context;
import android.content.Intent;
import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Context context = getApplicationContext();
        Intent explicitIntent = new Intent(context, LocationService.class);
        context.startService(explicitIntent);
    }


}

File: Challenges/5 - Justlisten/Solutions/Justlisten/app/src/main/java/com/example/justlisten/MainActivity.kt
package com.example.justlisten

import android.content.BroadcastReceiver
import android.content.IntentFilter
import android.os.Bundle
import androidx.activity.ComponentActivity


class MainActivity : ComponentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //Simply call the register receiver class
        val br: BroadcastReceiver = MyBroadcastReceiver()
        this.registerReceiver(br, IntentFilter("victim.app.FLAG_ANNOUNCEMENT"))

    }

}
